extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1136,12 +1153,12 @@ static public function DBtoID( $city_dbname, $master = false ) {
 	 */
 	static public function IDtoDB( $city_id, $master = false ) {
 
-		if ( ! self::isUsed() ) {
+		if ( ! static::isUsed() ) {
 			Wikia::log( __METHOD__, "", "WikiFactory is not used." );
 			return false;
 		}
 
-		$oRow = self::getWikiByID( $city_id, $master );
+		$oRow = static::getWikiByID( $city_id, $master );
 
 		return isset( $oRow->city_dbname ) ? $oRow->city_dbname : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)